README.md

```diff
@@ -41,6 +41,7 @@ This roadmap is used to track the progress on the project. If you add a feature
           * [x]  `while true {}`
           * [x]  `for foo in bar {}`
           * [ ]  `if foo {} else if bar {} else {}`
+          * [ ]  `match foo {bar => baz}`
```

Comment:
Maybe we can make these a bit more extensive so it's more welcoming to start working on just one of these items
```suggestion
          * [ ] if
            * [ ] `if foo {}`
            * [ ] `if foo {} else {}`
            * [ ] `if foo {} else if bar {} else {}`
          * [ ] match
            * [ ] `match foo { }`
            * [ ] `match foo { _ => {} }`
            * [ ] `match foo { bar => {} _ => {} }`
```